tests/testChecker.ml

```diff
@@ -736,6 +736,104 @@ let suite =
          (fun () -> Checker.entrypoint_update_operators (empty_checker, []))
     );
 
+    ("fa2 scenario" >::
+     fun _ ->
+       Ligo.Tezos.reset ();
+       let checker = empty_checker in
+
+       let initial_addr = Ligo.address_of_string "INIT_ADDR" in
+
+       (* mint some kit *)
+       Ligo.Tezos.new_transaction ~seconds_passed:10 ~blocks_passed:1 ~sender:initial_addr ~amount:(Ligo.tez_from_literal "100_000_000mutez");
+       let _, checker = Checker.entrypoint_create_burrow (checker, (Ligo.nat_from_literal "0n", None)) in
+       let max_kit = Checker.view_burrow_max_mintable_kit ((initial_addr, Ligo.nat_from_literal "0n"), checker) in
+
+       Ligo.Tezos.new_transaction ~seconds_passed:10 ~blocks_passed:1 ~sender:initial_addr ~amount:(Ligo.tez_from_literal "0mutez");
+       let _, checker = Checker.entrypoint_mint_kit (checker, (Ligo.nat_from_literal "0n", max_kit)) in
+
+       (* get some liquidity *)
+       Ligo.Tezos.new_transaction ~seconds_passed:10 ~blocks_passed:1 ~sender:initial_addr ~amount:(Ligo.tez_from_literal "0mutez");
+       let _, checker =
+         Checker.entrypoint_add_liquidity
+           ( checker,
+             ( ctez_of_muctez (Ligo.nat_from_literal "5_000_000n")
+             , kit_of_mukit (Ligo.nat_from_literal "5_000_000n")
+             , lqt_of_denomination (Ligo.nat_from_literal "5n")
+             , Ligo.timestamp_from_seconds_literal 999
+             )
+           ) in
+
+       (* initialize alice, bob and tom accounts *)
+       let tom_addr = Ligo.address_of_string "TOM_ADDR" in
```

Comment:
You might want to move this one into `test/testCommon.ml`, alongside `alice_addr`, `bob_addr`, and `leena_addr`, so that we don't have to redefine it in other places if we need more addresses.

Reply:
Hey, I've missed the introduction of `leena`! I'll just use `leena_addr` instead of `tom`.